Makefile

```diff
@@ -109,6 +109,10 @@ verify: tidy fmt vet generate manifests #HELP Verify all generated code is up-to
 fmt: #EXHELP Run go fmt against code.
 	go fmt ./...
 
+.PHONY: fix-fmt
+fix-fmt: $(GOLANGCI_LINT) #EXHELP Runs golangci-lint with --fix flag to fix linting issues.
+	$(GOLANGCI_LINT) run --fix $(GOLANGCI_LINT_ARGS)
+
```

Comment:
This sounds an awful lot like the `fmt` target.

Reply:
Is that a comment or a suggestion?

Reply:
merged the targets - is that what you wanted?